### PR TITLE
add terms of service

### PR DESCRIPTION
**Pull request by nikgraf**
Closes #42

### CODE REVIEW

File: pages/terms_of_service.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+These are our terms of service. In case you have any questions or concerns please reach out to us.
+
+## Refunds
+
+You registration is non-refundable once purchased. Cancellations of your travel and/or hotel reservations are your own responsibility. Please notice, that if you can't attend for any reason you are responsible to cancel your room and flights.
+
+## Transferring your Ticket
+
+You can transfer your ticket until the day before the conference. Transferring you ticket once the conference has started is not allowed.
```
Comment (ryyppy) [inline pages/terms_of_service.mdx]: Transferring you-r ticket once the conference...

File: pages/terms_of_service.mdx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## Cancellation
+
+The organisers can change the date of the event or cancel the event. If the event is cancelled any purchased ticket gets a full refund. If only a part e. g. a workshop is not taking place you get back the fee for the booked part, excluding the fee for the parts you still attend. Further claims against the organiser are not possible.
```
Comment (ryyppy) [inline pages/terms_of_service.mdx]: If the event is cancelled "," any purchased ticket gets...

File: components/footer.re [hunk 3]
```diff
@@ -31,6 +31,9 @@ let make = _children => {
             <li>
               <Link to_="/code-of-conduct/"> {"Code of Conduct" |> s} </Link>
             </li>
+            <li>
+              <Link to_="/terms-of-service/"> {"Terms of Service" |> s} </Link>
```
Comment (okonet) [inline components/footer.re]: I think we should add this link also to the tickets section.

File: pages/terms_of_service.mdx [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Terms of Service
+
+These are our terms of service. In case you have any questions or concerns please reach out to us.
```
Comment (okonet) [inline pages/terms_of_service.mdx]: Probably we should add "By purchasing a ticket and attending the conference you agreeing to the following terms"?

File: pages/terms_of_service.mdx [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+You can transfer your ticket until the day before the conference. Transferring your ticket once the conference has started is not allowed.
+
+## Cancellation
+
+The organisers can change the date of the event or cancel the event. If the event is cancelled, any purchased ticket gets a full refund. If only a part e. g. a workshop is not taking place you get back the fee for the booked part, excluding the fee for the parts you still attend. Further claims against the organiser are not possible.
+
+## Media
+
+Our team will record videos and take photos at the event. It may happen that you will be included. With your registration you grant us the right to use such material taken over the days. Still everyone has the change during registration to inform our team that she or he doesn't like to be photographed. We will do our best to not take any pictures of you. Regardless if you informed our team about your preference in case you have concerns about any photo or video snippet that we published please let us know and we will remove it.
```
Comment (okonet) [inline pages/terms_of_service.mdx]: > Still everyone has the change during registration to inform

`change` -> `chance` but I'd rewrite the whole as:

> If you don't want to be photographed, please inform us at the day of the conference at the registration desk.

Let's remove `We will do our best to not take any pictures of you.`

@nikgraf I'm not sure how we'll track these. Concat had different lanyards for such requests.

**Comment by okonet** [comment]
@nikgraf @ryyppy I've edited it a bit. Please review again.

**Comment by nikgraf** [comment]
@okonet looks good

**Comment by nikgraf** [comment]
I'm updating the TOS in Tito according to this

**Pull request merged**